May Heartbeat less main image (#2477)
iterative/dvc.org: content/blog/2021-05-15-may-21-dvc-heartbeat.md

﻿---
title: May '21 Heartbeat

> date: 2021-05-15

Reviewer: do we need to update dates?

description: |
  Monthly updates are here! We've hit 30 team members! MLOps learning 
  opportunities, tutorials with integrations, conference videos, Discord 
  server growth, and more!
descriptionLong: |
  This month you will find:

>   🚀 info on our growing team 

Reviewer 2: I think content on the team growth can be included after the other sections (in this list as well as in the details below) to highlight the other sections more.

  📖 MLOps learning opportunities,
  🔃 tutorials with integrations, 
  🎥 conference videos from our team members, 
  💥 Discord server growth, and 

>   more!

Reviewer: 💅 `more` - move it to the previous line?

grown so much that we now have two all-hands meetings! Affectionately called
UTC + and UTC -, these meetings honor all our different time zones while
allowing the other group to watch via recording when they are awake! You know
we're all about solving complicated problems. 💪🏼

![](/uploads/images/2021-05-15/team-map.png)_Our team_

## New Team Members

[Svetlana Sachkovskaya](https://www.linkedin.com/in/svetlana-sachkovskaya/) is
originally from Belarus, but is currently living in Poland. She has been a full
stack developer for over seven years. She loves traveling, meeting new people
and is excited to work on open source software. In her spare time you may find
her dancing the tango! 💃🏻 Welcome Sveta!

Exemplifying our diverse team in one fell swoop,
[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
has been working on DVC for a couple of years and is a long-standing contributor
to open source. He now joins us on the CML & Docs teams after completing his PhD
in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿

> [Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a

Reviewer: use the right Ş -> Şahin

> [Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a

Reviewer: DevRel -> DVC team (for now at least)

> techinical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,

Reviewer 3: ```suggestion
technical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
```

Reviewer 4: :smile: 
```suggestion
technical writer/ML enthusiast/AI dreamer/tutorial builder from Istanbul,
```

> Turkey. A self-described zelaot for technologies, Emre has worked in many

Reviewer 4: ```suggestion
Turkey. A self-described zealot for technologies, Emre has worked in many
```

Reviewer 5: Thank you for fixing my things :)) @casperdcl @skshetry

development/ML related projects and has been programming in Python since v. 1.7.
We are excited for Emre to bring you excellent technical content! ✍🏼

[Tapa Dipti Sitaula](https://www.linkedin.com/in/tapa-dipti-sitaula/) joins us
as a Senior Product Engineer from Nepal. She previously worked as a Principal
Engineer at a tech start up in India and has worked in various capacities in her
career from engineering to project management and communications. Her interests
include learning languages and breaking gender stereotypes. We're right there
with you Tapa! 🚀

## Open Positions

And we're still hiring!

[**Check out our three open roles**](https://weworkremotely.com/company/iterative)
for:

> - [**Senior Frontend Engineer**](https://weworkremotely.com/remote-jobs/iterative-senior-frontend-engineer)

Reviewer: link is broken, should be `front-end` (should we fix previous posts), please check other links

Next for your learning pleasure,
[Simone Scardapane](https://twitter.com/s_scardapane) is in the process of
fulfilling a "personal challenge" to create a PhD course for
[**Reproducible Deep Learning**](https://twitter.com/s_scardapane/status/1389240445788643329?s=20)
that includes the use of open source tools including our own DVC!
[Head to the link](https://github.com/sscardapane/reprodl2021) to star the repo
and cheer him on. We will be! 🙌🏼

![](/uploads/images/2021-05-15/reproducedl.png)_Reproducible Deep Learning PhD
Course_

[Simone Scardapane](https://twitter.com/s_scardapane) is in the process of
fulfilling a "personal challenge" to create a PhD course for
[**Reproducible Deep Learning**](https://twitter.com/s_scardapane/status/1389240445788643329?s=20)
that includes the use of open source tools including our own DVC!
[Head to the link](https://github.com/sscardapane/reprodl2021) to star the repo
and cheer him on. We will be! 🙌🏼

You see what I did there, right? **Reproducible**... **Deep Learning**.....  
Get it? Layers of wit people. I learned from the best! Just wanted to make sure
you were paying attention!

https://media.giphy.com/media/6ra84Uso2hoir3YCgb/giphy.gif

> # From the Community

Reviewer: Should we make it before Team Talks, before Other ML Ops ... ?

And now our beloved community! It's been another month full of community
goodness and we are grateful!

![Thank you](https://media.giphy.com/media/jmqWAjoxFCxJNHD2Kz/giphy.gif)

> ### Curvenote with DVC tutorials

Reviewer: Same - feels like we should be prioritizing community talks vs us?

### Discord Server Explosion

Our [Discord server](https://discord.com/invite/dvwXA2N) has exploded since last
month, up 30% in membership 😱, thanks in large part to a
[**Towards Data Science**](https://towardsdatascience.com/) post from
[Sara Metwalli](https://www.linkedin.com/in/sara-a-metwalli/) recommending
[**9 Discord Servers for Math, Python, and Data Science You Need to Join Today.**](https://towardsdatascience.com/9-discord-servers-for-math-python-and-data-science-you-need-to-join-today-34214b93d6b8)

Sara encourages readers to connect, learn and get inspired. 🚀 Thanks Sara!
We're on board with that! Rest assured our growing team is hard at work creating
content, improving tools and working on new tools 😶🤗 to continue to grow and
serve our MLOps community!

## Next Meetup

Don't miss our
[Meetup](https://www.meetup.com/DVC-Community-Virtual-Meetups/events/277245660)
May 27th at 3:00pm UTC, where we will hear from Puneeth Pai as mentioned above
👆🏽, as well as another user putting DVC and CML into action on his team, and
finally from David Ortega discussing CML pull requests! Bring your questions!
We're here to help!

## Tweet Love ❤️

> https://twitter.com/FullStackML/status/1389701264997445635?s=20

Reviewer: Do we have some community Tweets from the last month? :)

Author: Inserting Paige's Tweet.  Guess there are pluses to the delay...

  🔃 tutorials with integrations, 
  🎥 conference videos from our team members, 
  💥 Discord server growth, and 
  more!
picture: 2021-05-15/may21cover.png
author: jeny_defigueiredo
commentsUrl: https://discuss.dvc.org/t/dvc-may-2021-heartbeat/
tags:
  - Heartbeat
  - DVC
  - CML
  - MLOps
  - ML Repa
  - Andrew Ng
  - DAGsHub
  - Curvenote
---

# DVC News

We've hit 30 team members! Our team is distributed all over the world and has
grown so much that we now have two all-hands meetings! Affectionately called
UTC + and UTC -, these meetings honor all our different time zones while
allowing the other group to watch via recording when they are awake! You know
we're all about solving complicated problems. 💪🏼

![](/uploads/images/2021-05-15/team-map.png)_Our team_

## New Team Members

[Svetlana Sachkovskaya](https://www.linkedin.com/in/svetlana-sachkovskaya/) is
originally from Belarus, but is currently living in Poland. She has been a full
stack developer for over seven years. She loves traveling, meeting new people
and is excited to work on open source software. In her spare time you may find
her dancing the tango! 💃🏻 Welcome Sveta!

Exemplifying our diverse team in one fell swoop,

> [Casper da Costa-Luis](https://www.cdcl.ml) was born in Kenya, has an Indian
> Passport and lives in the UK. He has been working on DVC for a couple of years
> and contributing to open source since 2014. He now joins us full time on the CML
> team after completing his PhD in Medical Imaging. Fun facts about Casper include
> his becoming the U18 chess champion of Kenya when he was 14 and that he is a
> qualified SCUBA diver. 🤿

Reviewer 4: TMI

```suggestion
[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
has been working on DVC for a couple of years and is a long-standing contributor
to open source. He now joins us on the CML & Docs teams after completing his PhD
in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿
```

[Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a
techinical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
Turkey. A self-described zelaot for technologies, Emre has worked in many
development/ML related projects and has been programming in Python since v. 1.7.
We are excited for Emre to bring you excellent technical content! ✍🏼

> [Tapa Dipti Situala](https://www.linkedin.com/in/tapa-dipti-sitaula/) joins us

Reviewer 4: corerct teh spleling

```suggestion
[Tapa Dipti Sitaula](https://www.linkedin.com/in/tapa-dipti-sitaula/) joins us
```
